src/core/caniuse.js

```diff
@@ -0,0 +1,238 @@
+        caniuse.feature
+      }">caniuse.com</a>`;
+    }
+    resolve(content);
+  });
+  const definitionPair = hyperHTML.bind(document.createDocumentFragment())`
+    <dt class="caniuse-title" id="${`caniuse-${caniuse.feature}`}">
+      Can I Use this API?
+    </dt>
+    <dd class="caniuse-stats">${{
+      any: contentPromise,
+      placeholder: "Fetching data from caniuse.com...",
+    }}</dd>`;
+  headDlElem.appendChild(definitionPair);
+}
+
+/**
+ * Normalizes `conf.caniuse` object to hold normalized configuration
+ * @param {Object} conf   configuration settings
+ */
+function normalizeConf(conf) {
+  const DEFAULTS = {
+    maxAge: 24 * 60 * 60 * 1000, // 24 hours (in ms)
+    browsers: ["chrome", "firefox", "safari", "edge"],
+    versions: 4,
+  };
+  if (typeof conf.caniuse === "string") {
+    conf.caniuse = { feature: conf.caniuse, ...DEFAULTS };
+    return;
+  }
+  if (Array.isArray(conf.caniuse.browsers)) {
+    conf.caniuse.browsers = conf.caniuse.browsers
+      .map(b => b.toLowerCase())
+      .filter(isValidBrowser);
+  } else if (conf.caniuse.browsers !== "ALL") {
+    conf.caniuse.browsers = DEFAULTS.browsers;
+  }
+  Object.assign(conf.caniuse, DEFAULTS, { ...conf.caniuse });
+  function isValidBrowser(browser) {
+    if (BROWSERS.has(browser)) {
+      return true;
+    }
+    pub(
+      "warn",
+      `Ignoring invalid browser "\`${browser}\`" in [\`respecConfig.caniuse.browsers\`](https://github.com/w3c/respec/wiki/)`
+    );
+    return false;
+  }
+}
+
+/**
+ * promises to get content for canIUse table
+ * @param {Object} caniuseConf              normalized respecConfig.caniuse
+ */
+async function fetchAndCacheJson(caniuseConf) {
+  const { apiURL, feature, maxAge } = caniuseConf;
+  const url = apiURL
+    ? apiURL.replace("{FEATURE}.json", feature)
+    : `${GH_USER_CONTENT_URL}${feature}.json`;
+  // use data from cache data if valid and render
+  const cached = await cache.get(url);
+  if (cached && new Date() - cached.cacheTime < maxAge) {
+    return cached.stats;
+  }
+  // otherwise fetch new data and cache
+  const response = await window.fetch(url);
+  if (!response.ok) {
+    switch (response.status) {
+      case 404:
+        const msg = `Couldn't find feature "${feature}" on caniuse.com? Please check the feature key on [caniuse.com](https://caniuse.com)`;
+        pub("error", msg);
+    }
+    throw new Error(
+      `Response not ok from URL ${url} (HTTP Status ${response.status})`
+    );
+  }
+  const { stats } = await response.json();
+  // set it, and forget it (there is no recovery if it throws, but that's ok).
+  cache.set(url, { stats, cacheTime: new Date() }).catch(console.error);
+  return stats;
+}
+
+/**
+ * get HTML element for the canIUse support table
+ * @param  {Object} stats     CanIUse API results
+ * @param  {Object} conf      respecConfig.caniuse
+ */
+function createTableHTML(conf, stats) {
+  let browsers = conf.browsers;
+  if (conf.browsers === "ALL") {
```

Comment:
Did that because we don't have all the browsers in `BROWSERS` (that are in json response)
But as we are doing a `isValidBrowser` filter, this one is here is meaningless. 

Actually we should remove the `"ALL"` feature altogether as there really are too many browsers.